### PR TITLE
Reformat code

### PR DESCRIPTION
**Pull request by yagee-de**

### CODE REVIEW

File: mycore-wcms2/src/main/resources/META-INF/resources/modules/wcms2/test.html [hunk 1]
```diff
@@ -1,12 +1,11 @@
- <html>
-    <head>
-    	<title>Tree test</title>
-        <link rel="stylesheet" type="text/css" href="node_modules/dijit/themes/tundra/tundra.css">
-        <link rel="stylesheet" type="text/css" href="css/style.css">
-    </head>
-        <script type="text/javascript" src="node_modules/dojo/dojo.js"
-    djConfig="parseOnLoad: true, isDebug: true"></script>
-     <script type="text/javascript">
+<head>
```
Comment (yagee-de) [inline mycore-wcms2/src/main/resources/META-INF/resources/modules/wcms2/test.html]: Change looks not good to me

File: mycore-pi/src/main/resources/catalog.xml [hunk 2]
```diff
@@ -107,7 +107,7 @@
   <!-- Crossref 4.4.1 -->
   <system systemId="http://data.crossref.org/schemas/crossref4.4.1.xsd" uri="xsd/crossref/4.4.1/crossref4.4.1.xsd" />
   <system systemId="http://data.crossref.org/schemas/common4.4.1.xsd" uri="xsd/crossref/4.4.1/common4.4.1.xsd" />
-  <system systemId="http://data.crossref.org/schemas/common4.3.5.xsd" uri="xsd/crossref/4.3.5/common4.3.5.xsd" />
+  <system systemId="http://data.crossref.org/schemas/common4.3.5.xsd" uri="xsd/crossref/4.4.1/common4.3.5.xsd" />
```
Comment (yagee-de) [inline mycore-pi/src/main/resources/catalog.xml:110]: URI seems odd with 4.4.1 and 4.3.5

**Review by yagee-de — COMMENTED** [review]
Could you please correct?

File: mycore-pi/src/main/resources/catalog.xml (hunk 2 above)
Comment (rsteph-de) [reply mycore-pi/src/main/resources/catalog.xml:110]: That's O.K the old version xsd. lies in the folder of the new one

File: mycore-webcli/src/main/resources/META-INF/resources/modules/webcli/app/commands/commands.html [hunk 3]
```diff
@@ -1,6 +1,9 @@
 <li *ngFor="let command of commandList" class="dropdown-submenu" (mouseover)="onHoverSubmenu($event)">
-  <a class="dropdown-item" href="#" onclick='return false;'>{{command.name}}</a>
+  <a lass="dropdown-item" href="#" onclick='return false;'>{{command.name}}</a>
```
Comment (sebhofmann) [inline mycore-webcli/src/main/resources/META-INF/resources/modules/webcli/app/commands/commands.html:2]: This is a bug

**Pull request merged**